src/software/ai/hl/stp/skill/pass/pass_skill.hpp

```diff
@@ -0,0 +1,33 @@
+#pragma once
+
+#include "software/ai/hl/stp/skill/base_skill.hpp"
+#include "software/ai/hl/stp/skill/pass/pass_skill_fsm.h"
+
+template <bool SHOULD_CHIP>
+class BasePassSkill : public BaseSkill<PassSkillFSM, DribbleSkillFSM, PivotKickSkillFSM>
```

Comment:
needs docs. `SHOULD_CHIP` should be documented

Reply:
done